recipes/flex/all/conanfile.py

```diff
@@ -88,4 +94,11 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = ["fl"]
-        self.env_info.PATH.append(os.path.join(self.package_folder, "bin"))
+
+        bindir = os.path.join(self.package_folder, "bin")
+        self.output.info("Appending PATH environment variable: {}".format(bindir))
+        self.env_info.PATH.append(bindir)
+
+        lex_path = os.path.join(bindir, "flex")
```

Comment:
This would make the variable safe(r) to use in msys2 environments.
```suggestion
        lex_path = os.path.join(bindir, "flex").replace("\\", "/")
```

Reply:
@madebr How come it's not safe already?

Interesting, I see this code is sprinkled all over the CCI. Also in [documentation](https://docs.conan.io/en/latest/howtos/custom_generators.html?highlight=replace). If this is a common pattern, we could do better. Maybe a common method somewhere. Something like this?
~~~Python
def native_path(self, path):
    return path.replace("\\", "/") if tools.os_info.is_windows else path
~~~

Reply:
A path with forward slashes is not a Windows native path.
e.g. the Windows `more` utility (=Windows cousin of Linux' `cat`) does not support paths with forward slashes.

A utility function might be useful, but it might also come back to shoot us when that utility function gets extra features.